sdformat_mjcf/tests/mjcf_to_sdformat/test_add_mjcf_geometry_to_sdf.py

```diff
@@ -201,6 +201,18 @@ def test_no_type(self):
         self.assertEqual(radius, sdf_geom.sphere_shape().radius())
 
 
+class SurfaceTest(unittest.TestCase):
+    def test_friction(self):
+        mjcf_model = mjcf.from_path(
+            str(TEST_RESOURCES_DIR / "test_mujoco.xml"))
+
+        sdf_collision = geometry_conv.mjcf_collision_to_sdf(
+            mjcf_model.worldbody.geom[0])
+        self.assertIsNotNone(sdf_collision)
+
+        self.assertEqual(0.9, sdf_collision.surface().friction().ode().mu())
```

Comment:
Can you add one more assertion on a geom without `friction` explicitly set?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/86/commits/2292615d249f727e8f1150684158f57088384424